src/overcooked_ai_py/planning/planners.py

```diff
@@ -571,13 +581,17 @@ def _compute_plan_from_joint_graph(self, joint_start_state, joint_goal_state):
         
         Args:
             joint_start_state: pair of start (pos, or)
-            goal_statuses: pair of goal (pos, or)
+            joint_goal_state: pair of goal (pos, or)
         """
         assert self.is_valid_joint_motion_pair(joint_start_state, joint_goal_state), joint_goal_state
         # Solve shortest-path graph problem
         start_positions = list(zip(*joint_start_state))[0]
         goal_positions = list(zip(*joint_goal_state))[0]
         joint_positions_node_path = self.joint_graph_problem.get_node_path(start_positions, goal_positions)[1:]
+        if len(joint_positions_node_path) == 0:
+            # The cost will be infinite if there is no path
+            num_player = len(goal_positions)
+            return [], None, [np.inf] * num_player
```

Comment:
Maybe do the `try`/`except` approach I was suggesting in the `search.py` (?) file

Reply:
Addressed! Thanks for the wonderful suggestion!